src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -603,7 +603,7 @@ void RDMAConnectedSocketImpl::notify()
   int ret;
 
   ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  assert(ret == sizeof(i));
```

Comment:
@amitkumar50 
Yes that's what I meant in my comment `2` and `3`. What about `1` ?